client/src/components/Allocations/AllocationsTables.module.scss

```diff
@@ -0,0 +1,9 @@
+.root {
+  /* title */     tr > *:nth-child(1) { width: 13%; }
+  /* pi */        tr > *:nth-child(2) { width: 24%; }
+  /* team */      tr > *:nth-child(3) { width: 14%; }
+  /* systems */   tr > *:nth-child(4) { width: 12%; }
+  /* awarded */   tr > *:nth-child(5) { width: 12%; }
+  /* remaining */ tr > *:nth-child(6) { width: 12%; }
+  /* expires */   tr > *:nth-child(7) { width: 12%; }
+}
```

Comment:
__Explanation__
I have set widths for each column to avoid wrapping of cell content. And I have done so with compressed syntax. _N.B. There is a comment [at `table-layout: fixed`](https://github.com/TACC/Core-Portal/pull/350/files#diff-91d8b5279fe1f9ebde780c7326518a31d1819a9e26fe698b449bb9659c145287R7) about the solution that made widths required._

Reply:
This change occurs because flex is no longer used for table layout, so the `table-layout: fixed` we had now kicks in. (See [`InfiniteScrollTable.scss`](https://github.com/TACC/Core-Portal/blob/task/FP-385-section-comp-welcome-msg--sections/client/src/components/_common/InfiniteScrollTable/InfiniteScrollTable.scss) for the relevant code.)